README.md

```diff
@@ -0,0 +1,57 @@
+# BAHIS Data Pipeline
+
+This repository contains the data pipeline for the BAHIS project. It is a collection of scripts that download data from the BAHIS postgreSQL database, process it, and export it as csv files that can be used by the dashboard.
+
+## Local Installation
+
+First [install pipenv](https://pipenv.pypa.io/en/latest/install/). If you installed pipenv using `sudo apt install pipenv` and [getting this error](https://github.com/pypa/pipenv/issues/5133) install using `pip` instead.
+
+1. `pipenv install`
+2. copy data from `output` to `bahis-dash/exported_data/` if running the dashboard
+
+If you get an error when running install like this:
+
+```bash
+Resolving dependencies...
+✘ Locking Failed!
+```
+
+remove the `Pipfile.lock` file and run again.
+
+## Local development
+
+Run `pipenv run python pipeline.py`. (If you get a `Permission denied` error you may need to run as `sudo`)
+
+This will run a development server with hot reloading and other useful features.
+
+## Deployment
+
+To run the system in a local "deployment" you can use the Dockerfile with `docker build -t data . && docker run -p 80:80 -name bahis-data data:latest`. Note this will fail if `bahis_creds_file.cnf` is not valid, i.e. points to a local postgreSQL instance. You will also need to make sure the backup sql file is in the `input` directory.
```

Comment:
running the code `docker build -t data . && docker run -p 80:80 -name bahis-data data:latest` returns the error "unknown shorthand flag: 'n' in -name"
running `docker build -t data . && docker run -p 80:80 --name bahis-data data:latest` works

Reply:
'docker build -t data . && docker run -p 80:80'